File: Socket_program/clientudp.py
#!/usr/bin/env python3
import socket

def Main():
    host = '127.0.0.1'
    port = 50001

    s = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
    server = (host, port)

    message = input("Type your message: ")
    s.sendto(message.encode('utf-8'), server)

    data, server = s.recvfrom(1024)
    print("Received from server: " + data.decode('utf-8'))

    s.close()

if __name__ == '__main__':
    Main()

File: Socket_program/clientudp.c
#include <stdio.h>
#include <arpa/inet.h>
#include <fcntl.h>
#include <unistd.h>

int main()
{
    int soc; // n;
    char buffer[1024], msg[100];
    struct sockaddr_in addr; //

    soc = socket(PF_INET, SOCK_DGRAM, 0);
    
    addr.sin_family = AF_INET;
    addr.sin_port = htons(7891);
    addr.sin_addr.s_addr = inet_addr("127.0.0.1");
   
    printf("\nClient is connected to Server");
    printf("\nEnter message: ");
    scanf("%s",  msg);

    sendto(soc, msg, sizeof(msg), 0,(struct sockaddr*)&addr, sizeof(addr));
    close(soc);
    return 0;
}
